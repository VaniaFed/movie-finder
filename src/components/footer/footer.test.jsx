import renderer from 'react-test-renderer';
import { Footer } from './index';

describe('<Footer />', () => {
    const tree = renderer.create(<Footer />).toJSON();
    it('should be rendered with text "netfixroulette"', () => {
        expect(tree).toMatchSnapshot();
    });
});
