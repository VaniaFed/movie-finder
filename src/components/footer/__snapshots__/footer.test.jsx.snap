// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should be rendered with text "netfixroulette" 1`] = `
<footer>
  <h1
    className="logo"
  >
    netfixroulette
  </h1>
</footer>
`;
